tests/installation/installation_overview.pm

```diff
@@ -69,6 +69,12 @@ sub run {
         $self->deal_with_dependency_issues;
         assert_screen "inst-xen-pattern" if get_var('XEN');
         ensure_ssh_unblocked;
+        # Fail here if we have not graphical.target when we expect it
+        if (get_var('DESKTOP') && !check_var('DESKTOP', 'textmode')) {
+            select_console 'install-shell';
+            assert_script_run("grep 'set: graphical' /var/log/YaST2/y2log", fail_message => "Wrong systemd target detected, aborting");
```

Comment:
Will it work if we will check `/etc/systemd/system/default.target` file instead parsing logs?

Reply:
No, At that point the target is not set yet, it is done later during the install process.

Reply:
I see. Thanks for the explanation!